content/en/configuration/all.md

```diff
@@ -59,10 +59,26 @@ dataDir
 : (`string`) The designated directory for data files. Default is `data`. {{% module-mounts-note %}}
 
 defaultContentLanguage
-: (`string`) The project's default language key, conforming to the syntax described in [RFC 5646]. This value must match one of the defined language keys. Default is `en`.
+: (`string`) The project's default content language key, conforming to the syntax described in [RFC 5646]. This value must match one of the defined language keys. Default is `en`.
 
 defaultContentLanguageInSubdir
-: (`bool`) Whether to publish the default language site to a subdirectory matching the `defaultContentLanguage`. Default is `false`.
+: (`bool`) Whether to publish the default content language site to a subdirectory matching the `defaultContentLanguage`. Default is `false`.
+
+defaultContentRole
+: {{< new-in 0.150.0 />}}
+: (`string`) The site's default content role. Default is `guest`.
+
+defaultContentRoleInSubdir
+: {{< new-in 0.150.0 />}}
+: (`bool`) Whether to publish the default content role to a subdirectory matching the `defaultContentRole`. Default is `false`.
+
+defaultContentVersion
+: {{< new-in 0.150.0 />}}
+: (`string`) The site's default content version. Default is `TODO`.
+
```

Comment:
v1.0.0

See https://github.com/gohugoio/hugo/pull/14134

Reply:
This issue should now be correct/complete re. new things: https://github.com/gohugoio/hugo/issues/13776